en/documentation/index.md

```diff
@@ -71,6 +79,61 @@ The authoritative Ruby language documentation at [docs.ruby-lang.org][docs-rlo-e
 [RubyAPI.org][rubyapi-org]
 : Easily find and browse Ruby classes, modules, and methods.
 
+[ruby-doc.org][39]
+: online API documentation
+
+[DevDocs.io][40]
+: online API documentation
+
+[Ruby QuickRef][42]
+: The ruby quick reference
```

Comment:
```suggestion
: The Ruby quick reference
```

Reply:
I would add that this is for both the language and the standard library. Many references for other languages cover just one or the other.